### PR TITLE
docs: improve setting up documentation

### PR DESCRIPTION
**Pull request by G3root**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation

### Before this PR

{Please add a screenshot here}

### After this PR

{Please add a screenshot here}

### CODE REVIEW

File: apps/web/lib/env.ts [hunk 1]
```diff
@@ -17,8 +17,11 @@ export const env = () =>
 
       TINYBIRD_TOKEN: z.string(),
 
-      UNKEY_API_URL: z.string().url().default("https://api.unkey.dev"),
-      NEXT_PUBLIC_UNKEY_API_URL: z.string().url().default("https://api.unkey.dev"),
```
Comment (chronark) [inline apps/web/lib/env.ts:21]: I need to double check this doesn't break our prod

File: apps/agent/.env.example [hunk 2]
```diff
@@ -1,5 +1,15 @@
 
 TINYBIRD_TOKEN=
-DATABASE_DSN=
+
+DATABASE_DSN=USER:PASSWORD@tcp(HOST)/?tls=true
+
+# a random token which should be simillar to the value used in /apps/web/.env
```
Comment (rahulgpt) [inline apps/agent/.env.example:6]: *a random token that should be the same as in /apps/web/.env

Comment (chronark) [reply apps/agent/.env.example:6]: `openssl rand -base64 32` should work on linux/mac, not sure about windows

File: apps/web/lib/env.ts (hunk 1 above)
Comment (chronark) [reply apps/web/lib/env.ts:21]: I checked, we can safely change the defaults here

**Pull request merged**